torch/csrc/autograd/python_variable.cpp

```diff
@@ -771,7 +771,10 @@ static Tensor make_tensor_for_subclass_helper(
   Storage storage{
       Storage::use_byte_size_t{},
       size_bytes,
-      at::DataPtr{nullptr, options.device()},
+      // REVIEW: without the device change here,
+      // test_dtensor_save_load breaks. I'm not at all sure why it was
+      // OK before.
```

Comment:
based on test failures, changing the device to meta here clearly isn't correct . will go back.